### ISSUE TITLE
FEAT: Copy column to clipboard.

### ISSUE THREAD

**Issue by theOofDoctor** [beekeeper-studio/beekeeper-studio#1245]
**Is your feature request related to a problem? Please describe.**
Quite often when I'm querying data, I like to be able to copy all of the related primary keys so that I can use that on another table, or use it in an update statement (ex. `UPDATE user SET validated = NULL WHERE pk IN (...)`). Currently in beekeeper I can't find a way to copy multiple fields at the same time.

**Describe the solution you'd like**
SSMS (which is what my company currently uses) allows you to select the whole column, shift + click a portion of the column, or ctrl + click and select specific records in that column you'd like to copy.

**Describe alternatives you've considered**
The last 2 suggestions are more nice to haves, but I consider selecting the whole column a must for this use case.

**Additional context**
This is what happens in SSMS if you click on a column header: 
![image](https://user-images.githubusercontent.com/58712803/170574539-52940b7f-bcd2-4727-b844-e097a5ba3455.png)
From here you can copy that data and paste it back into the editor:
![image](https://user-images.githubusercontent.com/58712803/170574649-7e758bd8-6a7d-42bb-bc8d-5626174e3ee7.png)
![image](https://user-images.githubusercontent.com/58712803/170574622-a6920110-2ed1-4b7d-a8fb-2fdd5d17a58e.png)

You can shift + click to select a section of that column:
![image](https://user-images.githubusercontent.com/58712803/170574717-38d7e495-90c9-4ec4-a6e9-1f37f5bc3799.png)
From here you can do the same as above and copy the selected fields and paste them into your query editor

You can also ctrl + click to select specific fields in that column:
![image](https://user-images.githubusercontent.com/58712803/170574857-199d5a11-bc30-48ca-9b61-91a0350abe68.png)

Thanks for reading. Let me know if there's anything else I can clarify/help with.

**Comment by rathboma** [comment]
Yep! I'm planning on doing a round of changes to make all tables 'excel-like'. So that would include:

- Selecting columns
- Selecting rows
- Selecting an arbitrary range of cells

I'll probably do this in the tabulator project itself as I think there's an out dated PR with part of this implemented already

**Comment by theOofDoctor** [comment]
@rathboma Oh cool! Thanks for the quick reply. Glad to know it's been thought of. Should I close this issue then?
